docs/cep/query-guide/functions/sinkmapper/keyvalue.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: keyvalue (Sink Mapper)
+title: key-value (Sink Mapper)
 ---
 
 The `Event to Key-Value Map` output mapper extension allows you to convert Stream App events processed by gdn SP to key-value map events before publishing them. You can either use pre-defined keys where conversion takes place without extra configurations, or use custom keys with which the messages can be published.
 
-Syntax
+## Syntax
 
     CREATE SINK <NAME> WITH (map.type="keyvalue")
 
 ## Example 1
 
     CREATE SINK FooStream WITH (type='inMemory', topic='stock', map.type='keyvalue') (symbol string, price float, volume long);
```

Comment:
Let's replace `type-'inMemory'` by `type='stream'` here and on the other examples.